src/PFire.Core/Protocol/XFireAttributeFactory.cs

```
@@ -39,7 +42,8 @@ public XFireAttribute GetAttribute(byte type)
         {
             if (!_attributeTypes.ContainsKey(type))
             {
-                throw new UnknownXFireAttributeTypeException(type);
+                return new UnknownXFireAttribute() { TypeId = type };
```

Is this temporary? As mentioned above, I prefer to throw an exception here